src/main/java/taxi/model/Driver.java

```diff
@@ -50,11 +74,13 @@ public boolean equals(Object o) {
         Driver driver = (Driver) o;
         return Objects.equals(id, driver.id)
                 && Objects.equals(name, driver.name)
+                && Objects.equals(login, driver.login)
+                && Objects.equals(password, driver.password)
                 && Objects.equals(licenseNumber, driver.licenseNumber);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(id, name, licenseNumber);
+        return Objects.hash(id, name, licenseNumber, login, password);
     }
 }
```

Comment:
In common mistakes was a point about toString method(), I don't know if we really need it, but if you will override it remember not to use the password inside of it.